rbac/api/cross_access/model.py

```diff
@@ -32,7 +32,8 @@ class CrossAccountRequest(models.Model):
     """Cross account access request."""
 
     request_id = models.UUIDField(default=uuid4, editable=False, unique=True, null=False, primary_key=True)
-    target_account = models.CharField(max_length=15, default=None)
+    target_account = models.CharField(max_length=36, default=None)
+    target_org = models.CharField(max_length=36, default=None, null=True)
```

Comment:
Curious why the length is increased, is it in case the account/org id is longer?

Reply:
We did this because there is a possibility that the org_id will be a uuid format which are 36 characters long.

Reply:
👍